packages/block-library/src/post-terms/index.php

```diff
@@ -49,10 +49,48 @@ function render_block_core_post_terms( $attributes, $content, $block ) {
  * Registers the `core/post-terms` block on the server.
  */
 function register_block_core_post_terms() {
+	$taxonomies = get_taxonomies(
+		array(
+			'public'       => true,
+			'show_in_rest' => true,
+		),
+		'objects'
+	);
+
+	// Split the available taxonomies to `built_in` and custom ones,
+	// in order to prioritize the `built_in` taxonomies at the
+	// search results.
+	$built_ins         = array();
+	$custom_variations = array();
+
+	// Create and register the eligible taxonomies variations.
+	foreach ( $taxonomies as $taxonomy ) {
+		$variation = array(
+			'name'        => $taxonomy->name,
+			'title'       => $taxonomy->label,
+			/* translators: %s: taxonomy's label */
+			'description' => sprintf( __( 'Display the assigned %s' ), $taxonomy->label ),
```

Comment:
This isn't enough for good i18n. Quoting from an old [Calypso comment](https://github.com/Automattic/wp-calypso/pull/5346#issuecomment-218601883) of mine,

> the hardcoded line 'Would you like to create one?' […] doesn't work well with a lot of languages. A concrete example is gender — nouns like _page_ or _post_ may carry gender, and the pronoun one needs to reflect the gender. In Portuguese:
>
> - a post is _um artigo_, masculine; the hardcoded line would be: _Quer criar um?_
> - a page is _uma página_, feminine; the hardcoded line would be: _Quer criar uma?_
>
> German, French and indeed most Indo-European languages will have the same problem.

The same problem applies when considering taxonomies, which could differ in gender (e.g. [PT] _uma categoria_, _um produto_), or — in more inflected languages like Finnish — could be subjected to case inflection (e.g. [FI] nominative _kategoria_, partitive _kategoriaa_).

The proper way to create these strings would be to make them CPT labels... but that seems very heavy-handed for such a specific string. So, instead, I suggest crafting the strings in a way that frees them from inflection (e.g. something as unoriginal as _Display the assigned taxonomy: %s_).

Reply:
So many nuances with languages 😄 . I updated per your suggestion.